### PR TITLE
feat: admin panel

### PR DESCRIPTION
**Pull request by Enubia**
added react-router

moved admin things around

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,6 @@
 		"connect-mongo": "5.0.0",
 		"cookie-parser": "1.4.6",
 		"cors": "2.8.5",
```
Comment (stacysealky) [inline package.json:45]: discord is removed? @Enubia

Comment (Enubia) [reply package.json:45]: Yes the bot is gone already

**Review by stacysealky — APPROVED** [review]
Looks good! Awesome job

File: package.json (hunk 1 above)
Comment (stacysealky) [reply package.json:45]: This doesn't affect the discord webhooks notifications, right? Just making sure

Comment (Enubia) [reply package.json:45]: Nope the webhooks are not affected by this

**Pull request merged**